x/bank/client/cli/tx.go

```diff
@@ -60,3 +69,77 @@ func NewSendTxCmd() *cobra.Command {
 
 	return cmd
 }
+
+// NewMultiSendTxCmd returns a CLI command handler for creating a MsgMultiSend transaction.
+// For a better UX this command is limited to send one coin from one address to multiple addresses.
+func NewMultiSendTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "multi-send [from_key_or_address] [to_address_1, to_address_2, ...] [amount]",
+		Short: "Send funds from one account to multiple others.",
+		Long: `Send funds from one account multiple others.
+By default, sends the [amount] to each address of the list.
+Using the '--split' flag, the [amount] is split equally between the addresses.
+Note, the '--from' flag is ignored as it is implied from [from_key_or_address].
+When using '--dry-run' a key name cannot be used, only a bech32 address.
+`,
+		Args: cobra.MinimumNArgs(4),
```

Comment:
I am not sure if I agree, you always need the `from` and the `amount` as well as at least two receives. Otherwise, I think you should just use `send`.

Reply:
yeah, that's a point. In this case, we'll update the description to `Send funds from one account to two or more accounts` for better clarity.

Reply:
Sure thing, done 👍🏾